app/models/user.rb

```diff
@@ -23,6 +23,10 @@ class User < ApplicationRecord
   }
   validate :personal_access_token_validator
 
+  def admin?
+    Octobox.config.github_admin_ids.include?(github_id.to_s)
```

Comment:
Thoughts on `return true if Octobox.config.github_admin_ids.empty?`

Reply:
Or if it contains some keyword that means everyone?

Reply:
yea, something along those lines. It's always annoying to find github ids as they don't appear on profiles, etc. And we tend to default to open at Shopify, so it's nice to let anyone see the progress if they want :)